index.html

```diff
@@ -3,212 +3,212 @@
+        text: 'Hr'
+      }, {
+        count: 1,
+        type: 'day',
+        text: 'Day'
+      }, {
+        count: 1,
+        type: 'week',
+        text: 'Wk'
+      }, {
+        count: 1,
+        type: 'month',
+        text: 'Mo'
+      }, {
+        type: 'all',
+        text: 'All'
+      }],
+      inputEnabled: false,
+      selected: 1
+    },
 
-            title: {
-                text: title
-            },
+    title: {
+      text: title
+    },
 
-            exporting: {
-                enabled: false
-            },
+    exporting: {
+      enabled: false
+    },
 
-            xAxis: {
-                type: "datetime",
-                ordinal: false,
-            },
+    xAxis: {
+      type: "datetime",
+      ordinal: false,
+    },
 
-            series: []
-        }, options)
+    series: []
+  }, options)
 
-        var chart = null;
-        var seriesByName = {};
+  var chart = null;
+  var seriesByName = {};
 
-        var context = {
-            priorValues: {},
-            getSeries: function(asOf, name) {
-                var s = seriesByName[name];
-                if (!s) {
-                    s = chart.addSeries({
-                        name: name,
-                        data: []
-                    })
-                    seriesByName[name] = s;
-                }
-                return s;
-            }
-        }
+  var context = {
+    priorValues: {},
+    getSeries: function(asOf, name) {
+      var s = seriesByName[name];
+      if (!s) {
+        s = chart.addSeries({
+          name: name,
+          data: []
+        })
+        seriesByName[name] = s;
+      }
+      return s;
+    }
+  }
 
-        var ws = new WebSocket("wss://pure-journey-3547.herokuapp.com/stream/" + dimName + "/" + dimKey + "/" + statType + "/" + statName);
-        ws.onopen = function() {
-            console.log("WebSocket opened")
-        };
-        ws.onmessage = function(evt) {
-            var msg = evt.data;
-            var resp = JSON.parse(msg);
-            if (resp.succeeded) {
-                if (!chart) {
-                    // Lazily make the chart once we have some data
-                    $('#' + id).highcharts('StockChart', fullOptions);
-                    chart = $('#' + id).highcharts();
-                }
-                for (var i = 0; i < resp.intervals.length; i++) {
-                    var interval = resp.intervals[i];
-                    var asOf = interval.asOfSeconds * 1000;
-                    onData(context, asOf, interval.values);
-                }
-                chart.redraw();
-            } else {
-                console.log("Response was unsuccessful", resp["error"]);
-            }
-        };
-        ws.onclose = function() {
-            // websocket is closed.
-            console.log("WebSocket closed");
-        };
+  var ws = new WebSocket("wss://pure-journey-3547.herokuapp.com/stream/" + dimName + "/" + dimKey + "/" + statType + "/" + statName);
```

Comment:
It's just for internal usage and the statshub address is unlikely to be changed, so I think it's ok to hardcode here instead of through configuration.

Moreover, it will be pushed to https://lantern-dashboard.s3-website-us-east-1.amazonaws.com (through I don't know how to do it yet) as a static page eventually, we may not have a way to set environment variable for it.